sig-usability/charter.md

```diff
@@ -0,0 +1,52 @@
+# SIG Usability Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+The scope of SIG usability is the core end-user usability of the Kubernetes project. Thiis covers
+topics like user experience and accessibility. The goals of the SIG are to ensure that the Kubernetes
+project is maximaly usable by users of a variety of different backgrounds and abilities. Examples
+of efforts include internationalization and accessibility of documentation.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- usability of end-user facing experiences (error messages, end-to-end tasks, etc)
+- accessibility of Kubernetes community artifacts, examples include:
+   - internationalization of documentation in binaries and documentation
+   - color choices for people with color blind-ness
+   - ensuring compatability with screen reader technology
```

Comment:
Screen reader friendliness also applies in the context of terminal interfaces.

Reply:
Not sure I 100% follow.

The existing statement is trying to cover that we would consider this for terminal tools like `kubectl` is there something I could add here to make that more clear?

Reply:
I understood "community" artifacts to not include CLI binaries (rather, website and other documentation). However, I could have seen "binaries" mentioned in the previous bullet point.